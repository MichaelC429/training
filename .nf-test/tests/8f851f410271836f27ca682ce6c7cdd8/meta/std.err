[33mNextflow 25.04.1 is available - Please consider updating your version to it(B[m
