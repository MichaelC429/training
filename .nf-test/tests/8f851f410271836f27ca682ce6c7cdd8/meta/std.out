ERROR ~ Error executing process > 'SAMTOOLS_INDEX'

Caused by:
  Process `SAMTOOLS_INDEX` terminated with an error exit status (127)


Command executed:

  samtools index 'reads_son.bam'

Command exit status:
  127

Command output:
  (empty)

Command error:
  .command.sh: line 2: samtools: command not found

Work dir:
  /workspaces/training/.nf-test/tests/8f851f410271836f27ca682ce6c7cdd8/work/cc/9698402665b759d3eb1b6a591a3c6f

Tip: when you have fixed the problem you can continue the execution adding the option `-resume` to the run command line

 -- Check '/workspaces/training/.nf-test/tests/8f851f410271836f27ca682ce6c7cdd8/meta/nextflow.log' file for details
